content/technical-guide/installation/_index.md

```diff
@@ -83,7 +83,7 @@ browser to http://localhost:8080 to verify that the application is running.
 In addition to the components that ship with Cawemo, a few external systems are required for running it, which need to be set up separately.
 
 - Server with Linux operating system on `amd64` architecture
-- [PostgreSQL](https://www.postgresql.org/) – supported versions: 12, 13, 14, 15 (newer versions _may_ work as well)
+- [PostgreSQL](https://www.postgresql.org/) – supported versions: 12, 13, 14, 15, 16 (other versions _may_ work as well)
```

Comment:
I changed it to a more generic term, because version 16 is already the latest version.